src/components/player/player-sidebar.tsx

```diff
@@ -88,55 +89,76 @@ const NotesTab: React.FC = () => {
+      <div className="flex flex-col h-full">
+        <div className="flex-grow overflow-hidden">
+          <SimpleBar
+            forceVisible="y"
+            autoHide={false}
+            scrollableNodeProps={{
+              ref: scrollableNodeRef,
+              id: 'notes-tab-scroll-container',
+            }}
+            className="h-full overscroll-contain p-4"
+          >
+            <div className="space-y-3">
+              {cues.map((cue: VTTCue) => {
+                const note = cue.text
+                const active = player.activeMetadataTrackCues.includes(cue)
+                return (
+                  <div key={cue.text}>
+                    {active && <Element name="active-note" />}
                     <div
-                      onClick={() => {
-                        actions?.seek(cue.startTime)
-                        track('clicked cue in sidebar', {cue: cue.text})
-                      }}
-                      className="w-full cursor-pointer underline flex items-baseline justify-end pt-3 text-gray-900 dark:text-white"
+                      className={classNames(
+                        'text-sm p-4 bg-white dark:bg-gray-900 rounded-md shadow-sm border-2 border-transparent',
+                        {
+                          'border-indigo-500': active,
+                          '': !active,
+                        },
+                      )}
                     >
-                      <time className="text-xs opacity-60 font-medium">
-                        {convertTime(cue.startTime)}
-                      </time>
+                      {note && (
+                        <ReactMarkdown className="leading-normal prose-sm prose dark:prose-dark">
+                          {note}
+                        </ReactMarkdown>
+                      )}
+                      {cue.startTime && (
+                        <div
+                          onClick={() => {
+                            actions?.seek(cue.startTime)
+                            track('clicked cue in sidebar', {cue: cue.text})
+                          }}
+                          className="w-full cursor-pointer underline flex items-baseline justify-end pt-3 text-gray-900 dark:text-white"
+                        >
+                          <time className="text-xs opacity-60 font-medium">
+                            {convertTime(cue.startTime)}
+                          </time>
+                        </div>
+                      )}
                     </div>
-                  )}
-                </div>
-              </div>
-            )
-          })}
+                  </div>
+                )
+              })}
+            </div>
+          </SimpleBar>
         </div>
-      </SimpleBar>
+        <div className="flex-shrink-0 p-4">
+          <button
+            type="button"
+            onClick={(e) => {
+              e.preventDefault()
+              sendPlayerState('ADD_NOTE')
```

Comment:
instead of passing `send` this way, we should have a callback like `onAddNote` that is passed in so that the parent etc can pass in a callback and do `send` at that level instead of passing in a more powerful `send` function like this.

so here it would be 

```
if(isFunction(onAddNote){
onAddNote()
}
```

or `onAddNote` could default to `noop` so it'd always be a function.